server/src/main/java/io/unitycatalog/server/service/AuthService.java

```diff
@@ -167,17 +168,37 @@ public HttpResponse grantToken(
             String cookieTimeout =
                 ServerProperties.getInstance().getProperty("server.cookie-timeout", "P5D");
             Cookie cookie =
-                Cookie.secureBuilder(AuthDecorator.UC_TOKEN_KEY, accessToken)
-                    .path("/")
-                    .maxAge(Duration.parse(cookieTimeout).getSeconds())
-                    .build();
+                createCookie(
+                    AuthDecorator.UC_TOKEN_KEY,
+                    accessToken,
+                    "/",
+                    Duration.parse(cookieTimeout).getSeconds());
             responseHeaders.add(HttpHeaderNames.SET_COOKIE, cookie.toSetCookieHeader());
           }
         });
 
     return HttpResponse.ofJson(responseHeaders.build(), response);
   }
 
+  @Get("/logout")
+  public HttpResponse logout(HttpRequest request) {
+    String authorizationCookie =
+        request.headers().cookies().stream()
+            .filter(c -> c.name().equals(AuthDecorator.UC_TOKEN_KEY))
+            .map(Cookie::name)
```

Comment:
@creechy @xx-db should this be `Cookie::value` like the change we made in AuthDecorator?

Reply:
yes, this needs to be fixed in both places

Reply:
@sudharshanraja-db for reference, the change we made is in my[ UI PR here](https://github.com/unitycatalog/unitycatalog/pull/666/files#).

Reply:
It also looks like there are a lot of old commits included. I got myself into a similar situation and had to sync my forked repo with main repo, then reset HEAD to last commit.

Reply:
@jamieknight-db  yes i made the changes to use Cookie::value and also regarding reset to head noted going forward will keep this in mind